### DISCUSSION TITLE
Confused between broker.Position method

### DISCUSSION THREAD

**Discussion by nitishkumargupta** [rodrigo-brito/ninjabot#8] [Q&A] [answered]
Hello,

I am confused with the broker.Position method used in the code in example/strategy.go

What does it return?
Please explain about assetPosition and quotePosition.

I am not getting this.

**Comment by rodrigo-brito** [comment, marked as answer]
Hi @nitishkumargupta,

In the pair `BTCUSDT`, the asset is BTC and the quote is USDT (BTC is quoted in USDT), i.e. you are buying/selling BTC with the base coin in USDT.
Sometimes trades use other coins as quotes. For example ETHBTC, the asset is ETH and the quote is BTC.

Do you have a suggestion for a better name?

Investopedia uses `base` and `quote`: https://www.investopedia.com/terms/c/currencypair.asp

**Reply by nitishkumargupta** [reply]
@rodrigo-brito I was confused about it so I asked. I do not have any suggestions.

Thanks
